projects/farmworker-wallet-os.md

```diff
@@ -0,0 +1,120 @@
+We’ve built two products, using this technology. For organizations we’ve built Preparese Platform<sup>TM</sup>, it allows them to quickly develop and launch digital trust enabled services and programs. For farmworkers, we’ve developed Preparese Mobile<sup>TM</sup> which combines a verifiable digital profile with a suite of tools that simplify interactions with organizations on the platform. 
+
+ 
+The Preparese Platform<sup>TM</sup> is being used by 9 organizations and has 5 digital services in production. The most recent service launched is being used to process and distribute [$80 million](https://www.usda.gov/media/press-releases/2021/09/07/usda-invests-700-million-grants-provide-relief-farm-and-food) in one-time relief payments to over 125,000 workers, under the United States Department of Agriculture’s [Farm and Food Workers Relief Program](https://www.ams.usda.gov/services/grants/ffwr) (FFWR).  
+
+  
+The second product, Preparese Mobile<sup>TM</sup>, is designed around the unique needs of farmworkers. We’ve developed a react native mobile app that lets farmworkers store, manage, and exchange their verified information. Engagement with organizations is made easier with [DIDComm](https://didcomm.org/book/v2/)-based communication capabilities such as text and video chat.  
+
+ 
+One of the cornerstone technologies of the solution is an interoperable Mendix enabled digital wallet. The wallet components need to be able to engage with various non-profit, government, and philanthropic sources, each with their own technology stacks. For this reason, interoperability and working with open standard frameworks has been a priority. The project team members have participated in TrustOverIP Foundation, Hyperledger Aries JS Working Group, and various other communities, including the OpenWallet Foundation of late, to ensure we adhere to best practices and standards.       
+
+ 
+While currently focused on farmworkers, the Mendix digital wallet engine can be used to help others. There are many other underserved communities that could benefit greatly from the privacy, accessibility, and security digital wallets can provide. With this social impact purpose in mind and spirit of further collaboration, Entidad and the UFW Foundation propose to open source the digital wallet engine used in their Preparese<sup>TM</sup> solution. By making these resources available, we hope to facilitate a wide variety of social impact.  
+
+ 
+Additionally, the project allows us to bridge and advance the interest of both the OpenWallet Foundation and Mendix communities.  By collaborating with the Mendix community, we not only have the ability to grow the number of digital trust technology practitioners, we also tap into a community with an established customer base. By making digital wallets components that can be easily integrated into their existing solutions, the project can drive further adoption and usage of interoperable, secure and privacy preserving digital wallets. 
+
```

Comment:
It would be great to see a picture of what exists and what is going to be contributed. From reading this, I see the following components:
1. Mendix (low code solution)
2. Preparese Platform
3. Preparese Mobile app (React native)
4. Mendix enabled digital wallet
5. Mendix digital wallet engine

I am not sure if 3, 4, or 5 is what is intended to be contributed via this project. A picture would help clarify the true scope and the interfaces/interactions with Mendix so we can determine whether there will be any licensing concerns.

Reply:
Thanks for the suggestion. We've updated the project proposal with a  #[diagram](https://github.com/Entidad/farmworkerwallet/blob/main/Farmworker_WalletOS_OSS_Project.png) that captures the intended contributions for this project. It's a busy diagram but we hope it helps to clarify your concerns.